### PR TITLE
OOP school library: set up associations

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
- Set up associations between classes and objects. ✔

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def many_students(student)
+    @students << student
+    student.Classroom = self
+  end
+end
```
Comment (helman101) [inline classroom.rb]: - [x] I would recommend using a meaningful name for this function, what action are you doing when you call this function, I think that something like `add_student` is better 😄

File: person.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+require './corrector'
```
Comment (helman101) [inline person.rb:1]: - [x] Good job so far, but you forgot one project requirement for the `Person` class, it should have many rentals too, please add this 😄
    imagen

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @xJokeer 👋, 

Nice job so far with this project 😄, but there are still some changes to do.

## Required Changes ♻️

- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 2 above)
Comment (CarlosE-Feliz) [reply person.rb:1]: Sorry about it! I just did that part!

File: classroom.rb [hunk 3]
```diff
@@ -8,7 +8,7 @@ def initialize(label)
     @students = []
   end
 
-  def many_students(student)
+  def add_student(student)
     @students << student
     student.Classroom = self
```
Comment (o-rashad) [inline classroom.rb]: [OPTIONAL] I guess you meant to write `student.classroom` not `student.Classroom`

**Review by o-rashad — APPROVED** [review]
Hi @xJokeer 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classroom.rb (hunk 3 above)
Comment (CarlosE-Feliz) [reply classroom.rb]: oh! thanks!

**Pull request merged**